### PR TITLE
Implemented the bottom of the landing page (meet our team section)

### PR DESCRIPTION
**Pull request by kerry918**

### CODE REVIEW

File: src/components/landing_page/Profile.js [hunk 1]
```diff
@@ -1,9 +1,213 @@
...
 
 export default function Profile() {
+    const classes = useStyles();
     return (
-        <div>
-            
+        <div className={classes.root}>
+            <h1 className={classes.title}>Meet our Team</h1>
+            <p className={classes.desc}>We are a dedicated group of students who work toward to common 
+                goal of buliding the tech community at UW for you
```
Comment (jingyi2318) [inline src/components/landing_page/Profile.js:33]: Not sure if you meant "We are a dedicated group of students who work toward **the** common goal of buliding the tech community at UW for you" here

**Review by jingyi2318 — APPROVED** [review]
The landing page is really starting to come together! I left a small comment, but otherwise, LGTM!

**Review by eyhhuang — APPROVED** [review]
lgtm!

File: src/components/landing_page/Profile.js (hunk 1 above)
Comment (kerry918) [reply src/components/landing_page/Profile.js]: Thanks for catching that, I have changed the wording.

**Pull request merged**